books/free-programming-books-langs.md

```diff
@@ -809,6 +809,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Let's learn Go!](http://go-book.readthedocs.io/en/latest/)
 * [Practical Cryptography With Go](https://leanpub.com/gocrypto/read) - Kyle Isom
 * [Practical Go Lessons](https://www.practical-go-lessons.com) - Maximilien Andile
+* [Production Go](https://leanpub.com/productiongo) - Herman Schaaf and Shawn Smith (EPUB, HTML, PDF) (:construction: *in process*)
```

Comment:
we add an access note for leanpub - take a look at some of the other titles See also #6852